conans/client/tools/scm.py

```diff
@@ -36,6 +36,16 @@ def _check_repo(cmd, folder, msg=None):
 class SCMBase(object):
     cmd_command = None
 
+    @classmethod
+    def get_version(cls):
+        try:
+            out, _ = subprocess.Popen([cls.cmd_command, "--version"], stdout=subprocess.PIPE).communicate()
+            version_line = decode_text(out).split('\n', 1)[0]
+            version_str = version_line.split(' ', 3)[2]
+            return Version(version_str)
+        except Exception as e:
+            raise ConanException("Error retrieving Git version: '{}'".format(e))
```

Comment:
If this is SCMBase is not Git necessarily, right?

Reply:
good catch

Reply:
removed